themes/tiddlywiki/vanilla/base.tid

```diff
@@ -627,6 +627,10 @@ button.tc-untagged-label {
 
 .tc-page-controls {
 	margin-top: 14px;
+}
+
+.tc-page-controls > span > button,
+.tc-page-controls > span > span > button{
```

Comment:
CSS rules are processed form right to left. So everytime a button is handled, the browsers CSS processor needs to check if a button is somewhere in a span which is a child somewhere in a span, which covered by any element that happens to be classed: tc-page-controls. I don't like those rules. IMO they will break soon.